core-aam/index.html

```diff
@@ -11199,6 +11199,132 @@ <h3>Special Events for Menus</h3>
       </section>
     </section>
 
+    <section id="mapping_algorithms">
+      <h2>Algorithms</h2>
+      <p>
+        Some <abbr title="Application Programming Interfaces">APIs</abbr> provide
+        methods which require specific algorithms to be followed. The following
+        sections provide the algorithm mapping tables for these methods.
+      </p>
+      <h3>ARIANotifyMixin Algorithm Mapping Tables</h3>
+      <p>
+        The <code>ARIANotifyMixin</code> provides methods for announcing content
```

Comment:
Non-blocking: Is it accurate to say that this mixin provides “methods” (plural)?

```suggestion
        The <code>ARIANotifyMixin</code> provides a method for announcing content
```